### ISSUE TITLE
false positive of 2034 due to unquoted var 

### ISSUE THREAD

**Issue by pokeball99** [koalaman/shellcheck#700]
```bash
#false positive of 2034
B=1
jq .b[$B]
```
is fixed by
```bash
#false positive of 2034
B=1
jq .b["$B"]
```

**Comment by koalaman** [comment]
You're right. This is another issue due to #701.

Correct code in either case is `jq ".b[$B]"`, as both the quoted and unquoted versions will fail if you have a file `.b1` in the current directory. 

This issue will be fixed as part of #701, so I'll close this for now.
